### PR TITLE
Allow setting email subject

### PR DESCRIPTION
**Pull request by hluk**

### CODE REVIEW

File: reviewrot/__init__.py [hunk 1]
```diff
@@ -282,6 +287,9 @@ def parse_cli_args(args):
     parser.add_argument('--email', nargs="+",
                         default=None,
                         help='send output to list of email adresses')
+    parser.add_argument('--subject',
```
Comment (pbortlov) [inline reviewrot/__init__.py:290]: Please add DEFAULT_SUBJECT as a default argument here

**Review by pbortlov — CHANGES_REQUESTED** [review]

File: reviewrot/__init__.py (hunk 1 above)
Comment (hluk) [reply reviewrot/__init__.py:290]: The problem is that I did not find a way to check if user set the value on command line so the value from config should be used instead.

The correct way would be to get default values from configuration and then load from command line. But that would be a bigger refactoring.

Comment (hluk) [reply reviewrot/__init__.py:290]: I've removed the mention about the default here.

**Review by pbortlov — APPROVED** [review]

**Comment by mkosiarc** [comment]
From the first look PR looks good, I could take a deeper look and test it in a couple of days. But if you are satisfied @pbortlov feel free to merge this.

**Comment by pbortlov** [comment]
@mkosiarc please feel free to test it and merge it

**Review by mkosiarc — APPROVED** [review]

**Pull request merged**